planetarium/NineChronicles
# Restore guided quest combination indicator.

### Description

SSIA

### How to test

1. Create a character and proceed to combination guided quest.
2. Check if indicator is properly displaying.
3. Craft the indicated gear and check if it is updated properly.

### Required Reviewers

@planetarium/9c-dev @Namyujeong
